### PR TITLE
Add info about replacing a faulty cluster

### LINKED ISSUE #1908: Add info about replacing a faulty cluster
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior
The clustering guide (https://docs.sensu.io/sensu-go/5.14/guides/clustering/) should include instructions for how to replace a faulty cluster.

## Current Behavior
Information about replacing a faulty cluster is not included in https://docs.sensu.io/sensu-go/5.14/guides/clustering/

## Possible Solution
Add instructions for replacing a faulty cluster.

## Context
https://sensucommunity.slack.com/archives/C68LV5M9U/p1572977703143100

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Add instructions for replacing a faulty cluster in clustering guide.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/1908

## Review Instructions
@rgeniesse I'm not sure this is correct or even close...it doesn't make sense to me that it's not a sensuctl command.

### CODE REVIEW

**Comment by hillaryfraley** [comment]
May need to add additional steps/considerations that should be called out in this context.
- How do you know that you need to replace a faulty member?
- Where can I find additional info? Link to etcd operation documentation to help people when adding a member to an existing cluster doesn't solve the problem.

**Comment by hillaryfraley** [comment]
@rgeniesse are there other faulty cluster member indicators we should mention?

I fixed all the etcd links b/c they were all out of date. I can propagate these corrections in other versions before merging (after I have this content correct)

File: content/sensu-go/5.14/guides/clustering.md
Comment (rgeniesse) [inline content/sensu-go/5.14/guides/clustering.md]: So we'll need to add some commands above on how to remove the faulty member as well, as this section is all about how to replace a fault member.

This page already has those commands above. We'll probably want to show a faulty member using `sensuctl cluster member-list`, then the command to remove it `sensuctl cluster member-remove 2f7ae42c315f8c2d`. From there, we can talk about adding the new one and call out that the below configuration is for the new member only.

We can pair up and go over this more a bit later if needed with 5.15 the priority at the moment

Comment (hillaryfraley) [reply content/sensu-go/5.14/guides/clustering.md]: @rgeniesse your explanation makes sense. Let me try to fix it and see what you think. Thank you!

**Comment by hillaryfraley** [comment]
@rgeniesse I couldn't figure out how to use `sensuctl cluster member-list` to see cluster member health, so I tried to put together instructions using the other commands that are already there.

File: content/sensu-go/5.14/guides/clustering.md
Comment (rgeniesse) [inline content/sensu-go/5.14/guides/clustering.md]:
```suggestion
Second, delete the faulty cluster member. To continue this example, you will delete cluster member `backend-4` using its ID field:
```

File: content/sensu-go/5.14/guides/clustering.md
Comment (rgeniesse) [inline content/sensu-go/5.14/guides/clustering.md]: I am not quite sure of the language to use here, but we don't want to say "add the faulty cluster member back". While it is true you can use the same name, same IP for the new cluster member, in practice that server will be completely different; the old one is gone as there is something wrong with it.

So we need something along the lines of "Third, add a newly made server to the cluster. It can be named the same as the faulty one, with the same IP address and mostly the same configuration. Before adding, make sure to change the `etcd-initial-cluster-state` to `existing` in the `backend.yaml` configuration file."

File: content/sensu-go/5.14/guides/clustering.md
Comment (hillaryfraley) [inline content/sensu-go/5.14/guides/clustering.md]:
```suggestion
Third, add a newly created member to the cluster. You can use the same name and IP address as the faulty member you deleted, with one change to the configuration: specify the `etcd-initial-cluster-state` as `existing`.
```

Comment (hillaryfraley) [reply content/sensu-go/5.14/guides/clustering.md]: @rgeniesse what do you think of this revision?

Comment (rgeniesse) [reply content/sensu-go/5.14/guides/clustering.md]: Looks great @hillaryfraley

**Review by rgeniesse — APPROVED** [review]
Approved pending suggestion commit for the step to add a new member and copy to all versions.

**Pull request merged**
